help/en/html/doc/Technical/TechRoadMap.shtml

```diff
@@ -152,62 +168,39 @@
       You can follow the current status of these on the <a href="CI-status.shtml">CI Status
       page</a>.
-      <p>At some point, the Java version required by JMRI has to move forward. For example, Oracle
-      <a href=
-      "http://www.oracle.com/technetwork/java/javase/javaclientroadmapupdate2018mar-4414431.pdf">has
-      announced</a> that they'll stop providing standalone JRE installers by the end of 2020, by
-      which time JMRI distributions will have to contain the Java runtime components or it won't be
-      possible to run JMRI on newly-bought PCs (<a href="http://adoptopenjdk.net">will continue to
-      provide independent JREs</a>). That in turn might require tools like jlink from Java 9 or
-      later.</p>
-
-      <p>While we don't know with certainty when JMRI development will move past Java 8, we expect
-      that it will be during the Spring 2021 development releases, leading to the June 2021
+      <p>Java 17 is the current LTS release.  JMRI is built and tests are run on it too,
+      but we don't recommend using it due to some remaining compatibility issues.
+
+      <p>In the near future, the Java version required by JMRI will move forward. We expect
+      that it will be during the Spring 2022 development releases, leading to the June 2022
       production releases requiring Java 11 and therefore being called JMRI 5.0. There might be one
       or two JMRI updates and minor releases early in Spring that would still work with Java 8.</p>
 
       <h4>Java Release and Operating System Support</h4>
       More information on Java releases and the operating systems that support them is on a
       <a href="JVMCapabilities.shtml">separate page</a>.
+
       <h2>Migration Notes</h2>
 
       <p>This is a section of notes for various code migrations that are in progress or
       contemplated. These don't directly affect JMRI users in the way that Java version changes do,
       but they may effect external users of the JMRI libraries.</p>
 
       <ul>
-        <li>At some point, we have to migrate away from <a href="Help.shtml">JavaHelp and
-        JHelpDev</a>. At a minimum, we'll need to <a href=
-        "http://weblogs.java.net/blog/brinkley/archive/2004/11/javahelp_v20_02.html">replace the
-        renderer</a>. There are <a href=
-        "http://stackoverflow.com/questions/9900110/javahelp-viewers-or-alternatives">several
-        alternative help systems</a> available. <a href=
-        "http://www.oracle.com/technetwork/topics/index-083946.html">Oracle Help</a> might be a
-        good choice. DocBook as a tool for generating multiple documentation forms is also being
-        considered.
-        </li>
-
         <li>USB access technology has advanced a lot since support for some basic libraries was
         added to JMRI. At some point, we should replace those early library versions, but it will
         break some user scripts.</li>
```

Comment:
Do we have any idea on:
- How many users are affected?
- How big changes is required for the users to make their scripts work?

Would it be reasonable to do this change together with the move to Java 11, or do we want to do these changes separately?